app/_hub/moesif/kong-plugin-moesif/index.md

```diff
@@ -1,37 +1,49 @@
 ---
-name: Moesif API Insights
+name: Moesif API Analytics
 publisher: Moesif
 
-categories:
+categories: 
   - analytics-monitoring
+  - logging
```

Comment:
@lena-larionova are there any (filter or other) issues with having multiple category selections?

Reply:
No technical issues, but we've had bug reports filed for plugins that sit in two categories - people get confused because the plugin displays twice on the page. Ideally, we don't want to use two categories, and instead pick the one that applies the most.